x/multisig/exported/types.go

```diff
@@ -77,3 +78,13 @@ func (pk PublicKey) ValidateBasic() error {
 func (pk PublicKey) String() string {
 	return hex.EncodeToString(pk)
 }
+
+// GetECDSAPubKey returns the ECDSA public key
+func (pk PublicKey) GetECDSAPubKey() (ecdsa.PublicKey, error) {
+	btcecKey, err := btcec.ParsePubKey(pk, btcec.S256())
```

Comment:
```suggestion
	btcecKey := funcs.Must(btcec.ParsePubKey(pk, btcec.S256()))
```
should panic if it's not parsable.